planning/freespace_planner/include/freespace_planner/freespace_planner_node.hpp

```diff
@@ -110,6 +116,8 @@ class FreespacePlannerNode : public rclcpp::Node
   NodeParam node_param_;
   PlannerCommonParam planner_common_param_;
   AstarParam astar_param_;
+  RRTStarParam rrtstar_param_;
```

Comment:
it may be better to move to RRTStar and AstarSearch class, then the param is set in the class

Reply:
@yukkysaito 
> it may be better to move to RRTStar and AstarSearch class

Actually each planner class already has planner-specific parameter as it's member, thus we need to pass the param struct to the algorithm when constructing the algorithm. 

https://github.com/autowarefoundation/autoware.universe/blob/ae75c9bee27c8cc440faea26a3672bc2063eb7f4/planning/freespace_planning_algorithms/include/freespace_planning_algorithms/astar_search.hpp#L130


I guess your concern is about the node class having planner specific parameters as it's member. I understand that it's bit dirty because that means the planner-specific parameter is defined multiple place (node and algorithm clases). So, how about this modification? 
https://github.com/autowarefoundation/autoware.universe/pull/904/commits/fda0312299ad094464a95ef0f1decfa24434355c

by this change, the node class no longer have planner specific params inside.

Reply:
In the same context, I think it might be better to the node class not have planner_common_param_ as a member. 

So I added the following change 
https://github.com/autowarefoundation/autoware.universe/pull/904/commits/18f6d155a40791146b93d1137f80f6facbf348de